app/models/manageiq/providers/openstack/cloud_manager/cloud_volume_snapshot.rb

```diff
@@ -1,2 +1,138 @@
 class ManageIQ::Providers::Openstack::CloudManager::CloudVolumeSnapshot < ::CloudVolumeSnapshot
+  include SupportsFeatureMixin
+
+  supports :create_volume_snapshot
```

Comment:
so last time I spoke with @durandom, we decided to not define full name to support feature.

so it should be 
supports :create
supports :update
etc.

it't like that now for the most of the features and it should be the way forward